defs/bugs/openstack/octavia.yaml

```diff
@@ -0,0 +1,10 @@
+2008099:
+  input:
+    path: 'var/log/octavia/octavia-worker.log'
+  expr: '.+octavia.amphorae.drivers.haproxy.exceptions .+ Removing incomplete section ''peers'
+  hint: octavia.amphorae.drivers.haproxy.exceptions
+  raises:
```

Comment:
Since the bug is fixed in mentioned package version can you add a version criteria like in https://github.com/canonical/hotsos/pull/294/files.

Reply:
Done as new commit

Reply:
Squash them?